sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts_from_url.py

```diff
@@ -218,7 +218,7 @@ def test_receipt_url_pass_stream(self, **kwargs):
         with open(self.receipt_png, "rb") as receipt:
             with pytest.raises(ValueError) as e:
                 poller = client.begin_analyze_document_from_url("prebuilt-receipt", receipt)
-                assert str(e) == "'document_url' needs to be of type 'str'. Please see `begin_analyze_document()` to pass a stream."
+            assert str(e.value) == "'document_url' needs to be of type 'str'. Please see `begin_analyze_document()` to pass a byte stream."
```

Comment:
This is officially working, tested it out with the wrong message before pushing. It will catch any changes in the error message now